ROADMAP.md

```diff
@@ -0,0 +1,65 @@
+# Roadmap
+Last update July 5th, 2019
+
+## Next releases - Short-Term
+
+### v0.1.4
+
+#### Trial interruption/suspension/resumption
+
+Handle interrupted or lost trials so that they can be automatically resumed by running workers.
+
+#### Command line tools to monitor experiments
+
+See #125 
```

Comment:
C'est valide ça ? Il me semble que non...

Reply:
Indeed! I added a valid URL formatting for markdown.